dotnet/src/Connectors/Connectors.AzureOpenAI/RequestFailedExceptionExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Net;
+using Azure;
+
+namespace Microsoft.SemanticKernel.Connectors.AzureOpenAI;
+
+/// <summary>
+/// Provides extension methods for the <see cref="RequestFailedException"/> class.
+/// </summary>
+internal static class RequestFailedExceptionExtensions
```

Comment:
We may want to move this to internal utilities if it needs to be shared with the OpenAI connector

Reply:
Captured in the TODO list and will be working on in a separate PR.